README.md

```diff
@@ -205,6 +205,20 @@ print(1 + 2*3)! //7
 print(1+2 * 3)! //9
 ```
 
+## Factorials
+
+C implements Factorials with exclamation marks behind the terminating exclamation mark. These must be seperated by at least one space. 
+```Java
+const var nofac = 3!!!!!  // Just a very exiting declaration
+const var fac-1 = 3 ! !   // 6
+const var fac0 = 3!! !   // 6
+const var fac1 = 3! !!   // 48
+const var fac2 = 6! ! ! //2.601 218 944E1746
+const const fac3 = 4? ! !! //1 961 990 553 600
```

Comment:
Why after the question mark? There is no need for this. I think if the line is terminated by a question mark the exclamation marks should go before it.

Reply:
A question mark prints debug information. Here, it would print debug information about `4`,  but with your suggestion, it would print debug information about `4! !!`, which is totally different.

Reply:
But doesn't the question mark need to be at the end of the line, to then print debug information for the whole line?

Reply:
Hm. I tend to think it's more finegrained.

For example your comment could be rewritten as
> But doesn't the question mark need to be at the end of the line? To then print debug information for the whole line!
which would just output `maybe` for the debug.

But with your original comment, it could print `This deservers a deeper investigation about the objective.`

The debug specification isn't exactly clear about that. @TodePond?

Reply:
Hi if you're unsure with how ? works just chuck a ? on the end of it